install/windows.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: docs
+title: Windows Installation
+permalink: /docs/install/windows/
+---
+This page describes how to install Airsonic under Windows as Stand-alone WAR, without using Tomcat. It also contains information about how to setup Airsonic as a Windows Service and let it run under an user/service account.
+
+> **NOTE**: If you like to run Airsonic by using HTTPS, take a look at the proxy section about [IIS](/docs/proxy/iis/).
+
+##### Prerequisites
+
+Download the latest [Airsonic Release](https://github.com/airsonic/airsonic/releases), you need the `airsonic.war`.
+
+Install [OpenJDK](https://openjdk.java.net) or another Java implementation. If you're using vanilla Airsonic, make sure you use version 13 or below. For Airsonic Advanced with 11.0.0 or newer you need to have at least version 11, but 14 is fine as well. Make sure to add Java to your `PATH` variable.
+
+If you like to run Airsonic as a Windows Service, download the [Non-Sucking Service Manager (NSSM)](https://nssm.cc/download).
+
+##### Install Airsonic
+
+Before we start, doublecheck if Java is working as expected and you setup the `PATH` environment variable right. Open a command prompt or the PowerShell console and type `java -version`. This should print the installed Java version.
```

Comment:
`is working as expected and [...] the PATH environment variable [is setup correctly]` sounds better in English?